model/abstract_model.py

```diff
@@ -19,6 +19,18 @@ def mainloop(self) -> None:
     @abstractmethod
     def get_schedule(self) -> Schedule:
         pass
+    
+    @abstractmethod
+    def get_user_schedule(self) -> Schedule:
+        pass
+    
+    @abstractmethod
+    def change_current_schedule(self):
```

Comment:
What is this method supposed to do? We don't supply a schedule. What is it changed to?

Reply:
This method only changes between the two schedules that exist in the demo model. A user schedule and a algorithm based schedule. That's it's' purpose right now.

Reply:
Would a better name be for example:
`change_between_schedules()`
or
`change_between_existing_schedules()`

Reply:
I see. Maybe something like `toggle_algorithmic_schedule()`?